### ISSUE TITLE
CLAPI Add host categories

### ISSUE THREAD

**Issue by matthieu-robin** [centreon/centreon#4395]
Hello team,

Can we add  host categories when we add a host ?

Thanks in advance.

**Comment by querwin** [comment]
Hello,

You can add your host in a category, using this command : 

https://documentation.centreon.com/docs/centreon-clapi/en/latest/objects/host_categories.html#addmember-and-setmember

Regards,

**Comment by matthieu-robin** [comment]
Yep, I know, but it could be easier to add the host categories when we add a host. Everything in one command :-)
